frappe/public/js/frappe/form/layout.js

```diff
@@ -403,10 +478,11 @@ frappe.ui.form.Layout = class Layout {
 
 		return false;
 	}
-	focus_on_next_field (start_idx, fields) {
+
+	focus_on_next_field(start_idx, fields) {
 		// loop to find next eligible fields
-		for (var i = start_idx + 1, len = fields.length; i < len; i++) {
-			var field = fields[i];
+		for (let i = start_idx + 1, len = fields.length; i < len; i++) {
+			let field = fields[i];
 			if (this.is_visible(field)) {
```

Comment:
handle focus on next field under next tab break, i.e when tab on last field of the previous tab break, auto activate the next tab break and focus on the first field of the next tab break

Reply:
Done!